// webcoreui/astro.js
import AccordionComponent from './components/Accordion/Accordion.astro'
import AlertComponent from './components/Alert/Alert.astro'
import AspectRatioComponent from './components/AspectRatio/AspectRatio.astro'
import AvatarComponent from './components/Avatar/Avatar.astro'
import BadgeComponent from './components/Badge/Badge.astro'
import BannerComponent from './components/Banner/Banner.astro'
import BottomNavigationComponent from './components/BottomNavigation/BottomNavigation.astro'
import BreadcrumbComponent from './components/Breadcrumb/Breadcrumb.astro'
import ButtonComponent from './components/Button/Button.astro'
import CardComponent from './components/Card/Card.astro'
import CarouselComponent from './components/Carousel/Carousel.astro'
import CheckboxComponent from './components/Checkbox/Checkbox.astro'
import CollapsibleComponent from './components/Collapsible/Collapsible.astro'
import ConditionalWrapperComponent from './components/ConditionalWrapper/ConditionalWrapper.astro'
import CopyComponent from './components/Copy/Copy.astro'
import DataTableComponent from './components/DataTable/DataTable.astro'
import FlexComponent from './components/Flex/Flex.astro'
import FooterComponent from './components/Footer/Footer.astro'
import GridComponent from './components/Grid/Grid.astro'
import GroupComponent from './components/Group/Group.astro'
import IconComponent from './components/Icon/Icon.astro'
import InputComponent from './components/Input/Input.astro'
import KbdComponent from './components/Kbd/Kbd.astro'
import ListComponent from './components/List/List.astro'
import MasonryComponent from './components/Masonry/Masonry.astro'
import MenuComponent from './components/Menu/Menu.astro'
import ModalComponent from './components/Modal/Modal.astro'
import PaginationComponent from './components/Pagination/Pagination.astro'
import PopoverComponent from './components/Popover/Popover.astro'
import ProgressComponent from './components/Progress/Progress.astro'
import RadioComponent from './components/Radio/Radio.astro'
import RatingComponent from './components/Rating/Rating.astro'
import RibbonComponent from './components/Ribbon/Ribbon.astro'
import SelectComponent from './components/Select/Select.astro'
import SheetComponent from './components/Sheet/Sheet.astro'
import SidebarComponent from './components/Sidebar/Sidebar.astro'
import SkeletonComponent from './components/Skeleton/Skeleton.astro'
import SliderComponent from './components/Slider/Slider.astro'
import SpinnerComponent from './components/Spinner/Spinner.astro'
import SpoilerComponent from './components/Spoiler/Spoiler.astro'
import StepperComponent from './components/Stepper/Stepper.astro'
import SwitchComponent from './components/Switch/Switch.astro'
import TableComponent from './components/Table/Table.astro'
import TabsComponent from './components/Tabs/Tabs.astro'
import TextareaComponent from './components/Textarea/Textarea.astro'
import ThemeSwitcherComponent from './components/ThemeSwitcher/ThemeSwitcher.astro'
import TimelineComponent from './components/Timeline/Timeline.astro'
import TimelineItemComponent from './components/TimelineItem/TimelineItem.astro'
import ToastComponent from './components/Toast/Toast.astro'

export const Accordion = AccordionComponent
export const Alert = AlertComponent
export const AspectRatio = AspectRatioComponent
export const Avatar = AvatarComponent
export const Badge = BadgeComponent
export const Banner = BannerComponent
export const BottomNavigation = BottomNavigationComponent
export const Breadcrumb = BreadcrumbComponent
export const Button = ButtonComponent
export const Card = CardComponent
export const Carousel = CarouselComponent
export const Checkbox = CheckboxComponent
export const Collapsible = CollapsibleComponent
export const ConditionalWrapper = ConditionalWrapperComponent
export const Copy = CopyComponent
export const DataTable = DataTableComponent
export const Flex = FlexComponent
export const Footer = FooterComponent
export const Grid = GridComponent
export const Group = GroupComponent
export const Icon = IconComponent
export const Input = InputComponent
export const Kbd = KbdComponent
export const List = ListComponent
export const Masonry = MasonryComponent
export const Menu = MenuComponent
export const Modal = ModalComponent
export const Pagination = PaginationComponent
export const Popover = PopoverComponent
export const Progress = ProgressComponent
export const Radio = RadioComponent
export const Rating = RatingComponent
export const Ribbon = RibbonComponent
export const Select = SelectComponent
export const Sheet = SheetComponent
export const Sidebar = SidebarComponent
export const Skeleton = SkeletonComponent
export const Slider = SliderComponent
export const Spinner = SpinnerComponent
export const Spoiler = SpoilerComponent
export const Stepper = StepperComponent
export const Switch = SwitchComponent
export const Table = TableComponent
export const Tabs = TabsComponent
export const Textarea = TextareaComponent
export const ThemeSwitcher = ThemeSwitcherComponent
export const Timeline = TimelineComponent
export const TimelineItem = TimelineItemComponent
export const Toast = ToastComponent
